src/redux/rockets/rockets.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+import getRockets from '../api';
+
+const initialState = {
+  rockets: [],
+};
+
+const rocketSlice = createSlice({
+  name: 'rockets',
+  initialState,
+  extraReducers: (builder) => {
+    builder.addCase(getRockets.fulfilled, (state, action) => ({
+      ...state,
+      rockets: [...action.payload],
+    }));
+  },
+});
+
```

Comment:
[OPTIONAL]
Your code is working fine. That is great. But It would be nice if you could use the DUCK pattern for writing the reducers. Following the Standard will make your code easier to debug for other developers.